src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -211,6 +209,13 @@ std::string Notification::toJson(const std::string& attrsFormat)
     jh.addNumber("maxFailsLimit", this->maxFailsLimit);
   }
 
+  // FIXME PR: only if maxFailsLimit is in use failsCounter will be printed
+  // However, maybe we want this failsCounter in any case? It can be an useful statistic...
```

Comment:
Feedback welcome!

Reply:
Maybe is a good idea to print it for debugging the subscription, but in the case you want to display it, a huge value on `maxFailsLimit` can be set.